core/src/main/scala/kafka/coordinator/group/GroupMetadataManager.scala

```diff
@@ -670,12 +675,16 @@ class GroupMetadataManager(brokerId: Int,
                   case unknownKey =>
                     throw new IllegalStateException(s"Unexpected message key $unknownKey while loading offsets and group metadata")
                 }
+                messagesRead += 1
+                bytesRead += record.sizeInBytes()
               }
             }
+            batchesRead += 1
             currOffset = batch.nextOffset
           }
         }
 
+        info(s"Number of messages/bytes/batches read: $messagesRead/$bytesRead/$batchesRead for partition $topicPartition")
```

Comment:
Will this be too noisy if there are lots of partitions? Do we want to have the log output be for all partitions instead?

Reply:
So, you are saying we add up all items like messages/bytes/batches read across all partitions and print them for the total number of partitions? 
And what you are saying is true, if we have a lot of partitions, then logs can be flooded with just these lines... 
The only thought there is, sometimes it might be useful debugging information.. So, instead of info, if we print it at a debug level?

Reply:
We already log a message when the loading for a partition is completed: https://github.com/apache/kafka/blob/trunk/core/src/main/scala/kafka/coordinator/group/GroupMetadataManager.scala#L552. We could perhaps add more information to that one instead of adding a new one.

Reply:
ok.. @dajac @ijuma , let me know what you guys think about this PR..